_component_design/toast.md

```diff
@@ -150,26 +166,47 @@ In the informational toast notification example here, the height of the toast in
 
 <section class="static-section" markdown="1">
 
-## Behaviors
-
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Placement, animations, and stacking
+### Behaviors
+
+Ensure that toast supports the following behaviors:
+
+ - **Placement and stacking**: Place toasts in the top-right corner of the page, 32px beneath the navigation bar. If more than one toast notifications are triggered simultaneously, vertically stack them 32px apart, with newer toasts placed on top.
+ - **Animation and timing**: The toast fades into place over `400 ms`, remains in place for 10 seconds, and fades out over `400 ms`. If the user hovers over the toast while it is fading out, it immediately gains full opacity. Toasts persist indefinitely on cursor hover. When the cursor moves off the toast, the toast persists for three more seconds, and then fades over the normal `400 ms`.
```

Comment:
not 100% confident these are the correct names for the functions we use for the animation... checking with ryan...